staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -354,10 +378,39 @@ func (cfgCtlr *configController) digestConfigObjects(newPLs []*flowcontrol.Prior
 			errs = append(errs, errors.Wrap(err, fmt.Sprintf("failed to set a status.condition for FlowSchema %s", fsu.flowSchema.Name)))
 		}
 	}
+	cfgCtlr.addUpdateResult(currResult)
+
 	if len(errs) == 0 {
-		return nil
+		return suggestedDelay, nil
+	}
+	return suggestedDelay, apierrors.NewAggregate(errs)
```

Comment:
`apierrrors.NewAggregate` will return a nil upon empty arrays. so L383-385 can be removed.